package.json

```diff
@@ -54,10 +54,14 @@
         "scopeName": "inline-template.ng",
         "injectTo": [
           "source.ts"
-        ],
-        "embeddedLanguages": {
-          "source.html": "html"
```

Comment:
is this no longer needed?

Reply:
Oh yeah. I removed this because there was a bug to fix with the embedded languages, but I'll do it in another PR. added back